compiler/rustc_lint/src/unused.rs

```diff
@@ -1063,6 +1063,8 @@ impl UnusedDelimLint for UnusedParens {
                                 _,
                                 _,
                             ) if node.is_lazy()))
+                    && !(ctx == UnusedDelimsCtx::ReturnValue
```

Comment:
Presumably a similar error would happen with `break`, does that capture it?

Reply:
Looking at the code, it looks like `ExprKind::Break` isn't treated specially. Would it be worth adding a variant to `UnusedDelimsCtx`?

Reply:
> Looking at the code, it looks like `ExprKind::Break` isn't treated specially. Would it be worth adding a variant to `UnusedDelimsCtx`?

Thank you. I add BreakValue to UnusedDelimsCtx. Please review again.